### ISSUE TITLE
Studio: Optimize more iterations without starting from beginning

### ISSUE THREAD

**Issue by medakk** [SCIInstitute/ShapeWorks#1122]
After completing an optimization run, it would be useful to be able to run a few more optimization iterations(with different hyperparameters?) on the obtained shape model instead of starting from the beginning everytime.

**Comment by sheryjoe** [comment]
Akin to fine-tuning in deep learning.

**Comment by akenmorris** [comment]
Yeah, the only problem is that your end result is probably not reproducible.  Unless we are suggesting storing different stages of parameters.
